.vib/ruby/goss/ruby.yaml

```diff
@@ -0,0 +1,18 @@
+command:
+  check-installed-gems:
+    exec: gem list --local
+    exit-status: 0
+    stdout:
+      {{ range $module := .Vars.modules }}
+      - {{ $module }}
+      {{ end }}
+  check-ssl:
+    exec: ruby -rnet/https -e "Net::HTTP.get URI('https://bitnami.com')"
```

Comment:
As mentioned before, `bundle` and `rake` are also important binaries, I would check them as well.